README.md

```diff
@@ -1,6 +1,13 @@
-dp-frontend-renderer
+# dp-frontend-renderer
+
 ====================
 
+## Deprication
+
+The renderer is being depracted, the template and toml files are to be migrated to the relevant controller/service.
```

Comment:
maybe worth mentioning that the toml files are specifically the localisation files?

Reply:
also typo: depracted => deprecated